cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -119,9 +117,9 @@ type McmManager struct {
 
 type instanceType struct {
 	InstanceType string
-	VCPU         int64
-	MemoryMb     int64
-	GPU          int64
+	VCPU         resource.Quantity
+	MemoryMb     resource.Quantity
```

Comment:
We could change this variable name to `Memory` now . Same in `AWSInstanceType` and `AzureInstanceType` structs.

Reply:
Yes